rasa/nlu/model.py

```diff
@@ -70,7 +68,7 @@ def load(model_dir: Text):
             return Metadata(data, model_dir)
```

Comment:
Line 68 in this will also have to change (since the `metadata.json` is now included in a zipped file)

Reply:
The `model_dir` passed on to this method/class should always be unzipped. So, there should be no need for updating the method.